app/services/doc_auth/selfie_concern.rb

```diff
@@ -0,0 +1,41 @@
+module DocAuth
+  module SelfieConcern
+    extend ActiveSupport::Concern
+    def selfie_live?
+      portait_error = get_portrait_error(portrait_match_results)
```

Comment:
`portrait` is the TrueID naming scheme. I think it's because that's what TrueID chose to name their key. Source: `          true_id_product&.dig(:PORTRAIT_MATCH_RESULT)` in `app/services/doc_auth/lexis_nexis/responses/true_id_response.rb`.